### ISSUE TITLE
Missing an option to not remove empty rules.

### ISSUE THREAD

**Issue by nirdom** [elchininet/postcss-rtlcss#190]
Currently, the plugin removes empty rules, which conflict with other plugins/infrastructure.
Can you add an option not to remove such rules?

**Comment by elchininet** [comment]
Hi @nirdom,

At the moment the plugin removes empty rules that resulted from the process of removing their declarations to create the LTR and RTL rules. Something like this:

#### Input

```css
.test {
    left: 10px;
}
```

#### Output

```css
/* This rule is removed */
.test { 
}

/* This rule is created and the declaration is moved from the previous rule to here */
[dir="ltr"] .test {
    left: 10px;
}

/* This rule is created and the declaration is created flipping the declaration in the original rule */
[dir="ltr"] .test {
    right: 10px;
}
```

What you desire is having an option to avoid the deletion of the empty rules that remained from the process described before or to avoid removing other rules that were already empty in the original CSS and the plugin wrongly removes them?

If it is the first, then I can implement it with a new option as you requested, if it is the second it should not occur and a proper fix should be made to avoid it.

**Comment by elchininet** [comment]
Hi @nirdom,
Did you get a chance to check my last question?
Regards.

**Comment by elchininet** [comment]
Hi @nirdom,
I really don‘t know if you are still interested on this. But I was testing the library and found the issue that you mentioned here. So it should be already fixed in version `4.0.4` of the package.
Regards
